docs/examples/actions/basic.vue

```
@@ -0,0 +1,32 @@
+<script setup lang='ts'>
+import type { ActionsProps } from '@artmate/chat'
+import { Actions } from '@artmate/chat'
+
+import { ElMessage } from 'element-plus'
+import { RefreshRight, DocumentCopy } from '@element-plus/icons-vue'
```

ESLint

[error] Expected "@element-plus/icons-vue" to come before "element-plus".

(perfectionist/sort-imports)